sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1159,7 +1167,6 @@ public void close() throws IOException {
     @Nullable abstract String getTopic();
     @Nullable abstract Coder<K> getKeyCoder();
     @Nullable abstract Coder<V> getValueCoder();
-    abstract boolean getValueOnly();
```

Comment:
Either way please split into multiple commits (ideally multiple PRs, since these commits are not logically related)